docs/Contributing/API-for-contributors.md

```diff
@@ -3037,28 +3037,84 @@ _Available in Fleet Premium._
 
 `POST /api/v1/fleet/software/batch`
 
+This endpoint is asynchronous, meaning it will start a background process to download and apply the software and return a `request_uuid` in the JSON response that can be used to query the status of the batch-apply (using the `GET /api/v1/fleet/software/batch/{request_uuid}` endpoint defined below).
+
 #### Parameters
 
 | Name      | Type   | In    | Description                                                                                                                                                           |
 | --------- | ------ | ----- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| team_id   | number | query | The ID of the team to add the software package to. Only one team identifier (`team_id` or `team_name`) can be included in the request, omit this parameter if using `team_name`. Ommitting these parameters will add software to 'No Team'. |
```

Comment:
Just double-checking if deleting `team_id` is intentional?

Reply:
Yes. I only see `team_name` on the code.

Probably a copy/paste residue when copying `Parameters` from the batch/scripts API (which does have team_id and team_name).